ext/standard/tests/file/symlink_link_linkinfo_is_link_error2.phpt

```diff
@@ -5,9 +5,6 @@ Test symlink(), linkinfo(), link() and is_link() functions : error conditions -
 if (substr(PHP_OS, 0, 3) == 'WIN') {
     die('skip no symlinks on Windows');
```

Comment:
A bit late but: we do support both symbolic links and hard links on Windows since PHP 5.3.0, (ps. this is about hard links), perhaps it could be a thing for your test crusade @carusogabriel =P

Reply:
@KalleZ Thanks for the tip, I'll start for them maybe. As soon as I leave work, I'll prepare my windows Set up to prepare a PR like this one and #3416